apps/browser/src/tools/popup/send-v2/send-v2.component.ts

```diff
@@ -26,10 +26,12 @@ import { PopOutComponent } from "../../../platform/popup/components/pop-out.comp
 import { PopupHeaderComponent } from "../../../platform/popup/layout/popup-header.component";
 import { PopupPageComponent } from "../../../platform/popup/layout/popup-page.component";
 
-export enum SendState {
-  Empty,
-  NoResults,
-}
+export const SendStates = {
+  Empty: 0,
+  NoResults: 1,
+};
```

Comment:
⛏️ The object should be frozen so that any attempt to change it at runtime is prohibited.

```suggestion
export const SendStates = Object.freeze({
  Empty: 0,
  NoResults: 1,
} as const);
```